tests/test_core.py

```diff
@@ -1475,70 +1475,10 @@ def test_bigarray(self):
   def test_mod_globalstruct(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_mod_globalstruct')
 
-  @no_wasm_backend()
+  @no_wasm_backend('specific alignment semantics may be asmjs-specific?')
```

Comment:
I left it with the question mark because I wasn't really sure what was happening. Probably the prudent thing to do would be remove the comment from this PR, reinvestigate, and get a better answer.
---
Oh wait no it does use long doubles. https://github.com/kripken/emscripten/pull/4560/files#diff-5f4324fb90a9b59ee798f5edc4ea8965R10 Which would explain why the asm2wasm version outputs 24 and s2wasm outputs 32, because long doubles are still f64 in wasm_backend.
So, change the comment to *that*